src/gradata/daemon.py

```
@@ -337,6 +383,192 @@ def _handle_end_session(self) -> None:
+        if query.strip():
+            try:
+                with d._brain_lock:
+                    results = (
+                        d._brain.search(query)
+                        if hasattr(d._brain, "search")
+                        else []
+                    )
+                    if isinstance(results, list):
+                        context_parts = [str(r) for r in results[:5]]
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)